DEVELOPMENT.md

```diff
@@ -1,77 +1,138 @@
-- [Local builds](#local-builds)
-- [Command documentation](#command-documentation)
-- [Unit testing](#unit-testing)
-- [Acceptance testing](#acceptance-testing)
-  - [Installing Cartographer](#installing-cartographer)
-- [Debug Code Completion](#debug-code-completion)
+# Development
 
----
+This doc explains how to set up a development environment so you can get started
+[contributing](CONTRIBUTING.md) to `Tanzu apps plugin`. Also
+take a look at:
 
-## Local builds
+- [How to add and run tests](#testing)
+- [Iterating](#iterating)
 
-Building the CLI plugin locally requires both go 1.18 and the [Tanzu CLI](https://github.com/vmware-tanzu/tanzu-framework/tree/main/cmd/cli#installation) with the builder plugin installed.
+## Prerequisites
 
-Install the Tanzu builder and test plugins:
+Follow the instructions below to set up your development environment. Once you
+meet these requirements, you can make changes and
+[deploy your own version of apps plugin](#starting-apps-plugin)!
 
-```sh
-tanzu init
-```
+Before submitting a PR, see also [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-```sh
-tanzu plugin repo add -b tanzu-cli-admin-plugins -n admin -p artifacts-admin
-```
+### Install requirements
 
-```sh
-tanzu plugin install builder
+You must install these tools:
+
+- [`go`]: for compiling the plugin as well as other dependencies - 1.16+
+- [`tanzu`]: for adding the plugin
+- [`git`](https://help.github.com/articles/set-up-git/): for source control
+- [`tanzu builder plugin`] for compiling and publishing plugins
```

Comment:
would be good to explicitly state that this is a tanzu CLI plugin.
`tanzu builder plugin` should be `builder plugin for Tanzu CLI` ?

Also would be good to link to tanzu-framework docs for installing that plugin

Reply:
>tanzu-framework docs for installing that plugin

I tried to find it on their github docs but couldnt find anything like that.